css/style.css

```diff
@@ -0,0 +1,1154 @@
+  width: 13.1rem;
+  height: 4.8rem;
+  position: relative;
+  top: 1.8rem;
+  margin: auto;
+}
+
+.form-button:hover {
+  background: #ffbe90;
+  color: #3b2b2b;
+}
+
+.footer-container {
+  position: absolute;
+  top: 385rem;
+  width: 100%;
+  height: 7.2rem;
+}
+
+.footer-icon-list {
+  width: 30rem;
+  margin: 1.8rem auto;
+}
+
+.footer-icon-list ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  list-style: none;
+}
+
+.footer-icon {
+  width: 3.4rem;
+  height: 1.6rem;
+  font-size: 2.5rem;
+  color: #505f79;
+  margin-right: 1.3rem;
+}
+
+.footer-icon:hover {
+  color: #347dfa;
+}
+
+.footer-vectorimg {
+  width: 3.4rem;
+  height: 2.6rem;
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.footer-vectorimg:hover {
+  filter:
+    invert(35%) sepia(21%) saturate(653%) hue-rotate(179deg)
+    brightness(93%) contrast(84%);
+}
+
+.left-navbar a:hover {
+  color: blueviolet;
+}
+
+.right-navbar ul li a:hover {
+  color: blueviolet;
+}
+
+@media screen and (max-width: 1324px) {
+  html {
+    font-size: 50%;
+  }
+
+  .box-1 {
+    font-size: 2.5rem;
+  }
+
+  .work-section {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .boxes-3,
+  .boxes-4,
+  .boxes-5,
+  .boxes-6,
+  .boxes-7,
+  .boxes-8 {
+    opacity: 1;
+    background-size: cover;
+  }
+
+  .remaining-boxes-container {
+    margin-top: 0;
+  }
+
+  .about-me-container-box1 {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .about-me-container-box2 img {
+    width: 80%;
+    height: 80%;
+    top: 1rem;
+  }
+
+  .language-container {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .contact-section {
+    margin-right: auto;
+  }
+
+  .illustration-header img {
+    left: 99rem;
+  }
+
+  .about-me-container {
+    position: relative;
+    left: 11rem;
+  }
+}
+
+@media screen and (max-width: 1044px) {
+  html {
+    font-size: 40%;
+  }
+
+  .illustration-header img {
+    left: 99rem;
+  }
+
+  .about-me-container {
+    position: relative;
+    left: 11rem;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .illustration-header img {
+    left: 67rem;
+  }
+
+  .multipost-boxes-container {
+    margin-right: 6rem;
+  }
+
+  .about-me-container {
+    left: -3rem;
+  }
+}
+
+@media screen and (max-width: 785px) {
+  .multipost-boxes-container {
+    margin-right: 3rem;
+  }
+
+  .about-me-container {
+    left: -8rem;
+  }
+
+  .illustration-header img {
+    left: 59rem;
+  }
+}
+
+@media screen and (max-width: 765px) {
+  html {
+    font-size: 30%;
+  }
+
+  .right-navbar {
+    display: none;
+  }
+
+  .nav-bar {
+    display: flex;
+  }
+
+  .left-navbar {
+    width: 100%;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .wrap {
+    position: relative;
+    top: 3%;
+    right: 4%;
+  }
+
+  .menu {
+    width: 2rem;
+    height: 0.3rem;
+    background: white;
+    margin: 0.3rem 0;
+  }
+
+  .menu:hover {
+    background-color: blueviolet;
+  }
+}
+
+@media screen and (max-width: 575px) {
```

Comment:
- [ ] The website is not very responsive 😓 At some points there are some issues, for example at `575px` the headline looks like this 👇 

![image](https://user-images.githubusercontent.com/50721479/213795570-84c9424f-81c6-46ad-ac65-8858a5a49d15.png)

- [ ] And at `580px` the project cards look messy a little 😓 

![image](https://user-images.githubusercontent.com/50721479/213796011-0d14dff4-cb47-44c9-a20d-4f539a94f5a4.png)